### PR TITLE
[FR] Update `Article_styling_criteria\Formatting` and `Article_styling_criteria\Writing`

### PR DESCRIPTION
**Pull request by Ganondorf-1**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 1]
```diff
@@ -282,66 +277,50 @@ Les abréviations, les acronymes et les sigles doivent prendre une majuscule, sa
...
+### Citations
 
-Extrait du manuel de style de Wikipédia :
+Consultez les [conventions typographiques de Wikipédia concernant les citations](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Citation).
 
-> N'incluez la ponctuation terminale à l'intérieur des guillemets que si elle était présente dans le texte original, sinon placez-la après le guillemet fermant. Dans la plupart des cas, cela signifie qu'il faut traiter les points et les virgules de la même manière que les points d'interrogation : les placer à l'intérieur des guillemets s'ils ne s'appliquent qu'au matériel cité et à l'extérieur s'ils s'appliquent à l'ensemble de la phrase.
+N'incluez la ponctuation à l'intérieur des guillemets que si elle était présente dans le texte original, placez-la après le guillemet fermant dans le cas contraire. Dans la plupart des cas, cela signifie qu'il faut traiter les points et les virgules de la même manière que les points d'interrogation : les placer à l'intérieur des guillemets s'ils ne s'appliquent qu'au matériel cité et à l'extérieur s'ils s'appliquent à l'ensemble de la phrase.
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Writing/fr.md:320]: if this part isn't in the french wikipedia article, mention that it is translated from the english one above and leave the angle bracket for the blockquote

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 2]
```diff
@@ -282,66 +277,60 @@ Les abréviations, les acronymes et les sigles doivent prendre une majuscule, sa
...
 ### Virgule en série
 
 La virgule de série, également connue sous le nom de virgule d'Oxford ou de Harvard, doit être utilisée.
 
-### Em dash
+### Citations
+
+Consultez les [conventions typographiques de Wikipédia concernant les citations](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Citation).
+
+Les guillemets ne se ferment après la ponctuation que si celle-ci ne peut pas être séparée de la citation (si cette citation forme une phrase complète, par exemple).
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Writing/fr.md:320]: you still need to indicate that the information was actually taken from the article, for example "The page mentions that..." or "According to said Wikipedia article, ..."

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 3]
```diff
@@ -209,53 +212,45 @@ Le nom du jeu, osu!, ne doit pas être mis en majuscule ou en italique. La marqu
...
 
 Certains mots ont des variantes. Leur orthographe préférée doit être utilisée et est la suivante :
 
-- `creator` ou `mapper` au lieu de `beatmapper`.
-- `mapped` au lieu de `beatmapped`.
-- `BN` ou `Beatmap Nominators` lorsqu'il s'agit des *nominateurs de Beatmap*.
-- `sign in` au lieu de `log in`, sauf si le nom d'un bouton ou d'un lien indique le contraire.
-- `sign out` au lieu de `log out`, sauf si le nom d'un bouton ou d'un lien indique le contraire.
-- `register` au lieu de `sign up`, sauf si le nom d'un bouton ou d'un lien indique le contraire.
+- `créateur` ou `mappeur` au lieu de `beatmapper`.
```
Comment (Biokax) [inline wiki/Article_styling_criteria/Writing/fr.md:251]: soit on utilise `eur`, soit `er`. Utiliser les deux pour ce terme semble ambiguë

Comment (Ganondorf-1) [reply wiki/Article_styling_criteria/Writing/fr.md:251]: Effectivement j'ai peut-être pas fait gaffe pour `beatmapper`

**Review by Biokax — DISMISSED** [review]

**Review by Biokax — APPROVED** [review]

**Comment by TicClick** [comment]
leaving this for Walavouchey as he apparently knows a bit of French

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 4]
```diff
@@ -1,38 +1,35 @@
----
-outdated: true
-outdated_since: 3283bf7d3bfc0d724600cf6330cbc1645f455b3e
----
-
 # Rédaction
 
-*Pour les normes de mise en forme, voir : [Critères de style des articles/Mise en forme](../Formatting)*
+*Pour les normes de mise en forme, voir : [Critères de mise en forme d'article/Mise en forme](../Formatting)*
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Writing/fr.md:3]:
```suggestion
*Pour les normes de mise en forme, voir : [Critères de mise en forme d'article/Mise en forme](../Formatting)*\
```

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 5]
```diff
@@ -282,66 +277,60 @@ Les abréviations, les acronymes et les sigles doivent prendre une majuscule, sa
...
 
-Les expressions `tu` et `ton` sont à éviter. Le `je` ne doit pas être utilisé. Les termes `nous`, `il` ou `sa` et `elle` ou `son` ne doivent pas être utilisés (voir ci-dessus pour les autres termes).
+Les expressions `tu` et `ton` sont à éviter. Le `je` ne doit pas être utilisé. Les termes `nous`, `il` ou `son` et `elle` ou `sa` ne doivent pas être utilisés pour désigner une personne (voir ci-dessus pour les autres termes).
 
 ---
 
-[Les articles avec registres informels](#registre-des-langues) peut ignorer cette section ; cependant, il faut éviter de mettre des `I` dans les paragraphes.
-
-## Grammar and syntax
+[Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; cependant, il faut éviter de mettre des `Je` dans les paragraphes.
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Writing/fr.md:296]:
```suggestion
[Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; cependant, il faut éviter de mettre des `je` dans les paragraphes.
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**